stats/stats-server/src/config/read.rs

```diff
@@ -0,0 +1,26 @@
+use super::{json_config, toml_config};
+use std::path::PathBuf;
+
+pub fn read_charts_config(path: PathBuf) -> Result<toml_config::Config, anyhow::Error> {
+    let extension = path.extension();
+    if extension == Some(std::ffi::OsStr::new("json")) {
+        let json_config: json_config::Config = config::Config::builder()
+            .add_source(config::File::from(path))
+            .add_source(
+                config::Environment::with_prefix("STATS_CFG")
```

Comment:
```suggestion
                config::Environment::with_prefix("STATS_CHARTS_CONFIG")
```

Or just `STATS_CHARTS`.

`STATS_CFG` does not seem to indicate what that prefix does

Reply:
I decided to choose the shortest prefix. `STATS_CHARTS` is fine